docs/explanations/data_collection.rst

```diff
@@ -0,0 +1,86 @@
+What data does Canonical collect from Ubuntu Pro machines?
+**********************************************************
+
+For the purposes of delivering Ubuntu Pro services to customers in compliance
+with the terms of the Ubuntu Pro subscription, some system data is sent to
+Canonical servers. This data is sent via a couple different methods, depending
+on the service and the purpose of that particular data element.
+
+This document categorizes data collection by method of collection.
+
+APT Package Downloads
+=====================
+
+If you have any of the following services enabled, then the following data
+collection method will be in use whenever downloading packages for one of
+these services.
+
+- ``anbox-cloud``
+- ``cc-eal``
+- ``cis``
+- ``esm-apps``
+- ``esm-infra``
+- ``fips``
+- ``fips-preview``
+- ``fips-updates``
+- ``realtime-kernel``
+- ``ros``
+- ``ros-updates``
+- ``usg``
+
+Whenever you ``apt install`` a package from a pro service (or ``apt upgrade``
+to a version of a package from a pro service), ``apt`` will make a GET request
+to ``esm.ubuntu.com`` that includes the package name and version and HTTP
+basic auth credentials that are tied to the Ubuntu Pro Subscription.
+
+For example, installing the ``hello`` package from ``esm-apps`` will result in
+a request that looks something like this:
+
+.. code-block:: text
+   https://bearer:$resource_token@esm.ubuntu.com/apps/ubuntu/pool/main/h/hello/hello_2.10-2ubuntu4+esm1_amd64.deb
+
+This request is necessary to download the Pro update and it necessarily
```

Comment:
the "necessarily" after already saying "necessary" early in the sentence feels _unnecessary_ (lol). But beyond that, it also makes the sentence super clunky. I'd suggest rectifying this by dropping the "it necessarily" so that the sentence reads:
"This request is necessary to download the Pro update and includes the following data."